.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,10 +34,10 @@ Please check if your PR fulfills the following requirements:
 
 
 ## Does this PR introduce a breaking change?
-```
-[ ] Yes
-[ ] No
-```
+
+- [ ] Yes
+- [ ] No
```

Comment:
Same here

Reply:
Or, we can have this as a single checkmark (checked vs unchecked for breaking changes)

Reply:
Agree

Reply:
It should be:

- [x] Contains no breaking changes.

That way it stands out from a glance at the PR list, because if it contains breaking changes, it will have a missing mark on the task list.